mpf/platforms/system11.py

```diff
@@ -510,4 +536,4 @@ def disable(self):
 
     def timed_enable(self, pulse_settings: PulseSettings, hold_settings: HoldSettings):
         """Pulse and enable the coil for an explicit duration."""
-        self.overlay.driver_action(self.platform.driver, pulse_settings, hold_settings, self.side)
+        self.overlay.driver_action(self.platform_driver, pulse_settings, hold_settings, self.side, True)
```

Comment:
This `self.platform.driver` instead of `self.platform_driver` is a mistake on my part from the last PR!